dev-scripts/test-migration-e2e.py

```
@@ -0,0 +1,640 @@
+# ---------------------------------------------------------------------------
+
+
+def reset_everything():
+    """Wipe var/ncps, drop dev databases, recreate S3 bucket, flush Redis."""
+    log("reset: removing var/ncps", Y)
+    if os.path.exists(VAR_NCPS):
+        shutil.rmtree(VAR_NCPS, ignore_errors=True)
+
+    for engine in ("postgres", "mysql"):
+        log(f"reset: dropping {engine} database", Y)
+        subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)